frontend/src/components/Utils/time.js

```diff
@@ -32,6 +32,21 @@ export function timestampToDateFormatted(msTimestamp, timezone='America/New_York
   return moment.tz(parseFloat(msTimestamp), timezone).format('dddd, LL');
 }
 
+/**
+ * Format a timestamp (in milliseconds) into a pretty string with just the date, 
+ * not including the day of the week..
+ * Example: 'January 19, 1970'.
+ *
+ * @param {!number} msTimestamp Timestamp in milliseconds of desired date.
```

Comment:
See nullable comments in #101.

```suggestion
 * @param {number} msTimestamp Timestamp in milliseconds of desired date.
```

Reply:
These changes will get overwritten by those in #101.